packages/webhooks/lib/asyncAction.ts

```diff
@@ -0,0 +1,57 @@
+import type { LogContext } from '@nangohq/logs';
+import { metrics } from '@nangohq/utils';
+
+import { deliver, shouldSend } from './utils.js';
+
+import type { AsyncActionResponse, DBEnvironment, DBExternalWebhook, NangoAsyncActionWebhookBody } from '@nangohq/types';
+
```

Comment:
Note that we decided not to create a log operation for each webhook. For actions it could quickly become unmanageable to have 2 top level operation for each async actions. We can change that later based on customer feedback